war/src/main/webapp/css/style.css

```diff
@@ -22,6 +22,10 @@
  * THE SOFTWARE.
  */
 
+.item-icon-freestyleproject .icn {
+  background: url('../images/items/freestyle-48.png') center center no-repeat !important;
+}
```

Comment:
I think we can remove this CSS class and the png file because they can be retrieved using the new API.